test/cli/samples/sourcemap-inline/src/Main.html

```diff
@@ -1,9 +1,9 @@
-<p>Hello world!</p>
-
 <script>
-	export default {
-		onrender () {
```

Comment:
I stared at this for a while going "wtf is `onrender`". It wasn't until I went and did a search that I realized it's [deprecated syntax](https://github.com/sveltejs/svelte/blob/daa9344cbc75e54e2a1ce0f8f2f5938fd4e6fe2f/CHANGELOG.md#181).

Glad this is getting updated, feels weird that the tests were using it 🤔